File: 06-data-structures/arrays/if_array.py
places_in_dhaka = ["Lalbagh Fort", "Ahsan Manzil", "Dhaka University"]
city_name = "Dhaka"
current_temperature = 32.5
is_monsoon = True

def check_and_print_message(variable, name):
    if isinstance(variable,list):
        print(f"The variable {name} is an array.")
    else:
        print(f"The variable {name} is NOT! an array.")


check_and_print_message(places_in_dhaka,"places_in_dhaka")
check_and_print_message(city_name,"city_name")
check_and_print_message(current_temperature,"current_temperature")
check_and_print_message(is_monsoon,"is_monsoon")

File: 06-data-structures/arrays/if-array.js
// Create different variables, each containing either an array or a non-array value.
// Now use isArray to check if each variable is an array.
// Print a message to the console indicating whether each variable is an array or not.

const placesInDhaka = ["Lalbagh Fort", "Ahsan Manzil", "Dhaka University"];
const cityName = "Dhaka";
const currentTemperature = 32.5;
const isMonsoon = true;

function checkAndLogMessage(variable, name) {
  if (Array.isArray(variable)) {
    console.log(`The vaiable ${name} is an array.`);
  } else {
    console.log(`The vaiable ${name} is NOT! an array.`);
  }
}

checkAndLogMessage(placesInDhaka, "placesInDhaka");
checkAndLogMessage(cityName, "cityName");
checkAndLogMessage(currentTemperature, "currentTemperature");
checkAndLogMessage(isMonsoon, "isMonsoon");
